### ISSUE TITLE
Labelme aborts directly after trying to save my first shape on a Raspberry Pi 4b

### ISSUE THREAD

**Issue by augusteastoncalabria** [wkentaro/labelme#1348]
### Provide environment information

I'm using a Raspberry Pi 4 with 8gb RAM. FIrst found this error without a virtual environment, but the error persisted after making a virtual environment. Since finding the error, I have tried re-installing pyqt5 using 

pip3 install pyqt5 --config-settings --confirm-license= --verbose 

before installing labelme and have tried installing opencv-python-headless (I saw previous issues with these packages and labelme that I thought might be relevant to my situation). I am not well versed enough in Python to identify if there is an issue with the code in label_list_widget.py or if it is an issue with a package conflict or something of the sort with the Raspberry Pi OS. Thanks for any help, this would be a lifesaver to figure out. And I have to say thanks to the makers of labelme, this has been such a useful tool for my research!

Which python: /home/pi/.labelme/bin/python3
python version: Python 3.9.2
labelme version 5.3.1 and 5.2.1

### What OS are you using?

Raspbian GNU/Linux 11 (bullseye)

### Describe the Bug

I use labelme from the command line to open an image:
_labelme /home/pi/test-image.png_
Labelme opens with the image pulled up. I draw any kind of shape on it and save the polygon, and labelme immediately closes upon clicking save, with this error: 

Traceback (most recent call last):
  File "/home/pi/.labelme/lib/python3.9/site-packages/labelme/widgets/label_list_widget.py", line 46, in paint
    textRect = style.subElementRect(QStyle.SE_ItemViewItemText, options)
TypeError: subElementRect(self, element: QStyle.SubElement, option: Optional[QStyleOption], widget: Optional[QWidget]): not enough arguments
Aborted

### Expected Behavior

It saves the shape and allows me to draw more shapes and outputs a .json file once I save and exit. 

### To Reproduce

Use a Raspberry Pi 4b with Bullseye
Make a new virtual environment (optional, but I would like to use it in a virtual environment)
install labelme using pip and run it via command line
open an image, draw a shape, and save it

**Comment by wkentaro** [comment]
> *This was generated by AI during triage.*

## Triage Notes

Parking this in `needs-triage` for a maintainer decision rather than leaving it waiting on the reporter, because there is nothing left to ask.

**What we've established so far:**

- This is a duplicate of #809 (2020), which has the identical traceback: `subElementRect(self, QStyle.SubElement, QStyleOption, QWidget): not enough arguments` from `HTMLDelegate.paint` in `labelme/widgets/label_list_widget.py`. Same root cause, two different PyQt5 installs (Manjaro with labelme 4.5.6, Raspbian with 5.3.1). It is a PyQt5/sip binding quirk where some builds require the optional `widget` argument.
- That code path no longer exists. labelme dropped PyQt5 for PySide6 in v7.0.0 (commit `4d9abca2`), and `HTMLDelegate` was rewritten afterwards (commit `0ad9e691`). Its unit tests pass on current `main` (v7.0.4).
- So the reported failure mode is not reachable in any current version.

**Recommendation for @wkentaro:** close this as a duplicate of #809, and consider closing #809 itself as obsolete for the same reason. Flagging rather than acting, since closing is your call.
